group_vars/plantain/plantain_staging.yml

```diff
@@ -45,6 +50,6 @@ rails_app_vars:
   - name: APPLICATION_HOST_PROTOCOL
     value: '{{application_host_protocol}}'
   - name: SOLR_URL
-    value: 'http://lib-solr-staging.princeton.edu:8983/solr/pulfa3-staging'
+    value: "http://localhost:8983/solr/{{ plantain_solr_core }}"
```

Comment:
These aren't being set for the user `deploy`, I am working to diagnose and fix these now.